src/todo/todo.lua

```diff
@@ -147,7 +147,9 @@ function todo.update_current_task_label(player)
         if task.assignee == player.name then
             todo.log(serpent.block(task))
             todo.get_maximize_button(player).caption =
-                {"todo.todo_maximize_button_caption", {"todo.todo_list"}, task.task}
+                {"todo.todo_maximize_button_caption",
+                 {"todo.todo_list"},
+                 string.match(task.task, "[^\r\n]+")}
```

Comment:
This should not work with an empty first line. Can be fixed with "[^\r\n]*" instead.
Should be tested as I'm not sure of the lua regex comportment.